SCClassLibrary/Common/Core/Boolean.sc

```diff
@@ -18,9 +18,13 @@ Boolean {
 	keywordWarnings {
 		// turn on/off warnings if a keyword argument is not found
 		_KeywordError
+		^this.primitiveFailed
+	}
+	trace {
+		// this is only available in a special debugging version of the app
```

Comment:
I know this is style-consistent (see 5 lines up) but FWIW, I really think that function descriptions should go on the line _above_ the function declaration, not inside it.